roles/heimdall/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: "Get {{user}} uid"
+  shell: "id -u {{user}}"
+  register: uid
+
+- name: "Get {{user}} gid"
+  shell: "id -g {{user}}"
+  register: gid
+
+- name: Stop and remove any existing container
+  docker_container:
+    name: heimdall
+    state: absent
+    
+- name: Create heimdall directories
+  file: "path={{item}} state=directory mode=0775 owner={{user}} group={{user}}"
+  with_items:
+- /opt/heimdall

```

Comment:
The indentation needs fixing. 

Tip: You should test out the PRs before sending them up; to make sure it runs and words as expected.